gapic/java/java_gapic_pkg_repos.bzl

```diff
@@ -0,0 +1,276 @@
+# The versions which are most likely to be updated frequently
+_PROTOBUF_VERSION = "3.6.1"
+_GAX_VERSION = "1.32.0"
+_GAX_BETA_VERSION = "0.49.0"
+_GRPC_VERSION = "1.13.1"
+_COMMON_PROTOS_VERSION = "1.13.0-pre1"
```

Comment:
I don't think we want to depend on the pre version here

Reply:
Yes, I went too far with upgrading the protos version (`1.13.0-pre1` is the dependency of the generator, but the generated artifacts depend on `1.12.0`)